cms/search/tests/test_publishers.py

```diff
@@ -192,4 +193,4 @@ def test_publish_deleted_logs(self, mock_logger_info):
 
         msg_dict = last_call_args[2]
         self.assertIn("uri", msg_dict, "Payload dict missing expected key 'uri'")
-        self.assertEqual(msg_dict["uri"], self.information_page.url_path)
+        self.assertEqual(msg_dict["uri"], build_page_uri(self.information_page))
```

Comment:
This still doesn't confirm that it is now correct, as the logic is abstracted behind `build_page_uri`. If the logic for `build_page_uri` changes, the test would pass because we are not explicitly asserting what we expect. I don't see a unit test for `build_page_uri`, so let's add one to verify that it correctly removes `/home/` from the path.

Reply:
Followed up on Slack around the need for the `build_page_uri` utility function. As I feel like when I implemented all of this, I did it for a reason. But a quick poke with `page.url` vs `page.url_path`, `page.url` gives the URI without the `/home/` prefix. So I need to dig a bit more into the docs around this/ figure out why I did what I did/ if there is a reason. I found some reasons briefly looking but not sure if that's applicable for our use cases.

Not a blocker, but curious.

Reply:
I can't recall, `.url` behaves differently depending on whether you have a multi-site setup or not, as it may or may not find the current site, since it appends the root URL in those cases. So either way, you might need to support both cases, in which we may as well use .url_path. It's also best to call `.url` with the request object, which you don’t have access to in signals.

Let’s revisit this later; for now, let's focus on getting it fixed.

Reply:
> I can't recall, .url behaves differently depending on whether you have a multi-site setup or not, as it may or may not find the current site, since it appends the root URL in those cases. 

This is what I came across too yesterday @MebinAbraham